test/265_dns_reverse_test.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+. ./config.sh
+
+IP=10.2.0.67
+
+start_suite "Reverse DNS test"
+
+weave_on $HOST1 launch
+start_container_with_dns $HOST1 --name=c1
+
+for name in bar bar black sheep have you any wool; do
```

Comment:
Presumably  the many names are there because the failure scenario can be masked by randomness in the entries order. Needs a comment.